docs/build-docs.sh

```diff
@@ -22,7 +22,7 @@ bash docs/kedro-datasets-docs.sh
 if [ "$action" == "linkcheck" ]; then
   sphinx-build -c docs/ -ETan -j auto -D language=en -b linkcheck docs/build/ docs/build/html
 elif [ "$action" == "docs" ]; then
-  sphinx-build -c docs/ -ETa -j auto -D language=en docs/build/ docs/build/html
+  python -m sphinx.cmd.build -c docs/ -ETa -j auto -D language=en docs/build/ docs/build/html
```

Comment:
If I understand correctly this has no effect right? May I ask to drop this change to avoid a conflict with https://github.com/kedro-org/kedro/pull/2459 ? 😅

Reply:
Sure, it actually does have an effect on GitPod and locally on my machine, the previous command actually didn't run on either. That being said, I'm not confident on what is going on here, so I'll remove this.

Reply:
Oh it's interesting you say that because I tried to build the docs on Gitpod and nothing was working, the pip-installed binaries were going to some black hole or something. I'll open an issue or ask the team about that.